test/basic.html

```diff
@@ -206,6 +206,77 @@
         });
       });
 
+      suite('accessibility', function() {
+        var LEFT = 37;
+        var RIGHT = 39;
+        var tabs;
+
+        setup(function () {
+          tabs = fixture('basic');
+          ensureDocumentHasFocus();
```

Comment:
(from additional offline discussion) I pulled the call out of `setup` and placed it inside each test that actually needs the document to have focus. It still has to be there for a couple of cases though since the tests are checking for something that happens in response to the item being focused, not just that a particular item was focused.